modular_skyrat/modules/integrated_circuits/code/integrated_electronics/subtypes/output.dm

```diff
@@ -0,0 +1,496 @@
+		"ping"			= 'sound/machines/ping.ogg',
+		"synth yes"		= 'sound/machines/synth_yes.ogg',
+		"synth no"		= 'sound/machines/synth_no.ogg',
+		"warning buzz"	= 'sound/machines/warning-buzzer.ogg'
+		)
+	spawn_flags = IC_SPAWN_DEFAULT|IC_SPAWN_RESEARCH
+
+/obj/item/integrated_circuit/output/sound/beepsky
+	name = "securitron sound circuit"
+	desc = "Takes a sound name as an input, and will play said sound when pulsed. This circuit is similar to those used in Securitrons."
+	sounds = list(
+		"creep"			= 'sound/voice/beepsky/creep.ogg',
+		"criminal"		= 'sound/voice/beepsky/criminal.ogg',
+		"freeze"		= 'sound/voice/beepsky/freeze.ogg',
+		"god"			= 'sound/voice/beepsky/god.ogg',
+		"i am the law"	= 'sound/voice/beepsky/iamthelaw.ogg',
+		"insult"		= 'sound/voice/beepsky/insult.ogg',
+		"radio"			= 'sound/voice/beepsky/radio.ogg',
+		"secure day"	= 'sound/voice/beepsky/secureday.ogg',
+		)
+	spawn_flags = IC_SPAWN_RESEARCH
+
+/obj/item/integrated_circuit/output/sound/medbot
+	name = "medbot sound circuit"
+	desc = "Takes a sound name as an input, and will play said sound when pulsed. This circuit is often found in medical robots."
+	sounds = list(
+		"surgeon"		= 'sound/voice/medbot/surgeon.ogg',
+		"radar"			= 'sound/voice/medbot/radar.ogg',
+		"feel better"	= 'sound/voice/medbot/feelbetter.ogg',
+		"patched up"	= 'sound/voice/medbot/patchedup.ogg',
+		"injured"		= 'sound/voice/medbot/injured.ogg',
+		"insult"		= 'sound/voice/medbot/insult.ogg',
+		"coming"		= 'sound/voice/medbot/coming.ogg',
+		"help"			= 'sound/voice/medbot/help.ogg',
+		"live"			= 'sound/voice/medbot/live.ogg',
+		"lost"			= 'sound/voice/medbot/lost.ogg',
+		"flies"			= 'sound/voice/medbot/flies.ogg',
+		"catch"			= 'sound/voice/medbot/catch.ogg',
+		"delicious"		= 'sound/voice/medbot/delicious.ogg',
+		"apple"			= 'sound/voice/medbot/apple.ogg',
+		"no"			= 'sound/voice/medbot/no.ogg',
+		)
+	spawn_flags = IC_SPAWN_RESEARCH
+
+/obj/item/integrated_circuit/output/sound/vox
+	name = "ai vox sound circuit"
+	desc = "Takes a sound name as an input, and will play said sound when pulsed. This circuit is often found in AI announcement systems."
+	spawn_flags = IC_SPAWN_RESEARCH
+
+/obj/item/integrated_circuit/output/sound/vox/Initialize()
+	.= ..()
+	sounds = GLOB.vox_sounds
+	extended_desc = "The first input pin determines which sound is used. It uses the AI Vox Broadcast word list. So either experiment to find words that work, or ask the AI to help in figuring them out. The second pin determines the volume of sound that is played, and the third determines if the frequency of the sound will vary with each activation."
+
+/obj/item/integrated_circuit/output/text_to_speech
+	name = "text-to-speech circuit"
+	desc = "Takes any string as an input and will make the device say the string when pulsed."
+	extended_desc = "This unit is more advanced than the plain speaker circuit, able to transpose any valid text to speech."
+	icon_state = "speaker"
+	cooldown_per_use = 10
```

Comment:
after messing around with circits id sudgest moving this down from 1 second to 0.1 second. as its mostly used for local translator. while chat spam may seem like a issue text to radio and lets everyone hear already sends the messages as up to 0.1